### PR TITLE
Add Bidog info and log

### PR DESCRIPTION
**Pull request by Binancedogofficial**
3000 TPT transfer address
https://bscscan.com/tx/0xd7c0196ee63c5f155bdc942dc8d4974eefa385194bf3f6a47efc196821c998b4

### CODE REVIEW

File: bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+ "name": "Binancedog",
+ "website": "https://www.binancedog.net/",
+ "description": "🐶Binance dog 🐶is a deflationary token which is centered by the community and driven by the internal force. 
+",
+ "explorer": "https://bscscan.com/address/0x265f638eab14dd43d80bb71aae3289b1c6d46a3f",
```
Comment (Bidogofficial) [inline bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json:6]: "explorer": "https://bscscan.com/address/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f",

Comment (Bidogofficial) [inline bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json:6]: "explorer": "https://bscscan.com/address/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f",

File: bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+ "name": "Binancedog",
+ "website": "https://www.binancedog.net/",
+ "description": "🐶Binance dog 🐶is a deflationary token which is centered by the community and driven by the internal force.
```
Comment (Bidogofficial) [inline bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json:4]: "description": "🐶Binance dog 🐶is a deflationary token which is centered by the community and driven by the internal force.",

File: bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+ "name": "Binancedog",
+ "website": "https://www.binancedog.net/",
+ "description": "🐶Binance dog 🐶is a deflationary token which is centered by the community and driven by the internal force. 
+",
```
Comment (Bidogofficial) [inline bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json:5]: delete

File: bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+ "name": "Binancedog",
+ "website": "https://www.binancedog.net/",
+ "description": "🐶Binance dog 🐶is a deflationary token which is centered by the community and driven by the internal force. 
+",
+ "explorer": "https://bscscan.com/address/0x265f638eab14dd43d80bb71aae3289b1c6d46a3f",
+ "type": "BEP20",
+ "symbol": "Bidog",
+ "decimals":9,
+ "status": "active",
+ "id": "0x265f638eab14dd43d80bb71aae3289b1c6d46a3f"
```
Comment (Bidogofficial) [inline bsc/0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f/info.json:11]: "id": "0x265f638eAb14dD43D80bB71AAe3289B1c6d46A3f"

**Comment by Bidogofficial** [comment]
Sorry I forget to transfer it into Checksum address I have change it in reviewer please accept it

**Review by Bidogofficial — CHANGES_REQUESTED** [review]

**Comment by guoxintongsam** [comment]
Please refer to the tutorial, and re-submit your files in the correct format. Please submit your logo in the correct size.

**Comment by jiasutijiao** [comment]
专业提交logo！审核失败的走加速审核小时内包过！价格显示问题！QQ:3579750034

**Pull request closed without merging**